Source/DataSources/GeoJsonDataSource.js

```diff
@@ -627,7 +626,6 @@ define([
             },
             set : function(value) {
                 defaultFill = value;
-                defaultFillMaterialProperty = new ColorMaterialProperty(defaultFill);
```

Comment:
It probably does not catch it because `defaultStrokeWidthProperty` is "used" on line 613.

In any case, I will remove `defaultStrokeWidthProperty` and `defaultStrokeMaterialProperty`